test/Removal.t.sol

```diff
@@ -259,7 +256,7 @@ contract Removal_release_listed is UpgradeableMarket {
       0
     );
     assertEq(_removal.balanceOf(address(_market), REMOVAL_ID_FIXTURE), 1);
-    _removal.release(address(_market), REMOVAL_ID_FIXTURE, 1);
+    _removal.release(REMOVAL_ID_FIXTURE, 1);
     assertEq(
       _removal.balanceOf(_namedAccounts.supplier, REMOVAL_ID_FIXTURE),
       0
```

Comment:
should we add a test case that covers the situation where a removal's balances are spread across the original supplier, the market contract, and the certificate contract, and see that `release` correctly burns from all three owners?

Reply:
def. tracking on #306